src/app/profile/payment-methods/payment-methods.component.js

```diff
@@ -54,14 +54,14 @@ class PaymentMethodsController {
     this.$rootScope.$on( SignOutEvent, ( event ) => this.signedOut( event ) );
 
     this.loading = true;
-    this.loadPaymentMethods();
-    this.loadDonorDetails();
```

Comment:
It used to call the load methods twice, once in $onInit and once on the session enforcer signin callback. @Omicron7 this is the question I asked you on Hipchat: Is it safe to remove the $onInit ones and rely only on the session enforcer? What would you suggest/what have you been doing elsewhere?